src/pages/Possibility/Possibility.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+	<www-page>
+		<router-view />
+	</www-page>
+</template>
+
+<script>
+import WwwPage from '@/components/WwwFrame/WwwPage';
+
+export default {
+	components: {
+		WwwPage
+	},
+	beforeCreate() {
+		const todaysDate = new Date();
+		const day = todaysDate.getDate();
+		const month = todaysDate.getMonth() + 1; // getMonth is 0 based
```

Comment:
@ryan-ludwig please look into making this switch at midnight PDT rather than basing it on system time.